katalog/harbor/distributions/full-harbor/ingress.yml

```diff
@@ -25,34 +25,55 @@ spec:
     - http:
         paths:
           - path: /
+            pathType: Prefix
             backend:
-              serviceName: portal
-              servicePort: 80
+              service:
+                name: portal
+                port:
+                  number: 80
```

Comment:
I'd suggest using the port names instead of the numbers when possible for easier reading and maintenance